src/m_cheat.c

```diff
@@ -403,13 +403,13 @@ static void cheat_notarget()
     plyr->message = "Notarget Mode OFF";
 }
 
-static void cheat_tst()
+static void cheat_tst(cheatarg_t arg)
 { // killough 10/98: same as iddqd except for message
-  cheat_god();
+  cheat_god((cheatarg_t){.i = 0});
```

Comment:
C89 doesn't support designated initializers, but I think we can move on to at least C99.

Reply:
How would this look in order to be C89 compliant?

Reply:
Like this?
https://github.com/fabiangreffrath/woof/blob/b12fa74d13228e54a20323eedfe5543d3a8a6550/src/m_misc.c#L115-L120

Reply:
Yes, but we still use `stdint.h` types, maybe I should remove this macro.

Reply:
So, by including `stdint.h` we have already left C89 terrain towards C99? Well, then I think it's time to let go. :wink: